readme.md

```diff
@@ -39,4 +39,4 @@ npm install -g bower gulp
 
 ## Extend it
 
-If you want to add additional features i recommend using the [generator-nodes](https://www.npmjs.com/package/generator-nodes) Yeoman generator (shameless plug!)
\ No newline at end of file
+If you want to add additional features i recommend using the [generator-nodes](https://www.npmjs.com/package/generator-nodes) Yeoman generator (shameless plug!)
```

Comment:
Not sure, why this got marked - must be a spaces Github online editor have removed.